docs/quickstart/www/analytics/init_data.php

```diff
@@ -1,20 +1,33 @@
 <?php
-
-// Setup to load the necessary classes from the example directory
-require_once(__DIR__ . '/../../bootstrap.php');
-
+/**
+     * Copyright (c) 2021 Learnosity, MIT License
+     *
+     * Basic example of pulling information from the Learnosity cloud using Data API.
+     */
+
+// Setup to load the necessary classes from the example directory, 
+// and set up variables related to user access.
+require_once __DIR__ . '/../../../../bootstrap.php';
+$config = require_once __DIR__ . '/../config.php';
 use LearnositySdk\Request\DataApi;
 use LearnositySdk\Request\Remote;
 
 $itembank_uri = 'https://data.learnosity.com/v1/itembank/items';
 
+// Public & private security keys required to access Learnosity APIs and
+// data. These keys grant access to Learnosity's public demos account, 
+// loaded from a configuration file on line 13.
+// Learnosity will provide keys for your own private account.
+$consumerKey = $config['consumerKey'];
+
 $security_packet = [
-    'consumer_key'   => 'yis0TYCu7U9V4o7M',
+    'consumer_key'   => $consumerKey,
     'domain'         => 'localhost',
 ];
 
-// XXX: The consumer secret should be in a properly secured credential store, and *NEVER* checked in in revision control
-$consumer_secret = '74c5fd430cf1242a527f6223aebd42d30464be22';
+// XXX: The consumer secret should be in a properly secured credential store, 
+// and *NEVER* checked into version control.
```

Comment:
We no longer need this warning here, but we should have it in the `config.php` instead.

Reply:
You can use the same blurb as the other files https://github.com/Learnosity/learnosity-sdk-php/pull/60/files#diff-459e34bbdd78631b275577a339dcd42ee59fc18752e529e9e42622d68f8be722R24-R29